src/OdasStream/odas_stream.py

```diff
@@ -0,0 +1,96 @@
+import subprocess
+import json
+import time
+import math
+import os
+import sys
+from threading import Thread
+from PyQt5.QtCore import QObject, pyqtSignal
+
```

Comment:
A new line between Python's default packages and  packages from the Internet would be nice.